standard/docs/en/schema/merging.md

```diff
@@ -51,106 +52,163 @@ If the record is complete with versioning information, then the versioning secti
 
 .. jsoninclude:: ../examples/merging/merge-award-2.json
    :jsonpointer: /releases
-   :expand: releases, award
+   :expand: releases, tag, awards
    :title: awardTwo
 
 ```
 
 ```eval_rst
 
 .. jsoninclude:: ../examples/merging/merged.json
-   :jsonpointer: 
-   :expand: records, compiledRelease, tender, award, tag
+   :jsonpointer:
+   :expand: records, compiledRelease, tag, tender, awards
    :title: record
 
 ```
 
 ```eval_rst
 
 .. jsoninclude:: ../examples/merging/versioned.json
-   :jsonpointer: 
-   :expand: records,versionedRelease, tender, award, tag
+   :jsonpointer:
+   :expand: records, versionedRelease, tag, tender, awards
    :title: versioned
 
 ```
 
 </div>
 
-## Merging rules
+## Merging specification
 
-Order all the releases that share an `ocid` by their release `date`. Starting from the oldest release (old), compare it with the next oldest release (new), and apply the following rules.
+### Discarded fields
 
```

Comment:
Suggest adding a brief explanation of why these fields are discarded. It *should* be obvious to people who know the standard enough to follow the merging advice, but worth addressing explicitly.

Reply:
Added in 50ee7f9